addons/armaos/functions/fnc_computer_playSoundStart.sqf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Plays a 3D computer start sound on the position of a given computer object.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ *
+ * Results:
+ * None
+ */
+ 
+ params ["_computer"];
+
+_filename = "z\ae3\addons\armaos\sounds\ComputerStartSound.ogg";
```

Comment:
private

Reply:
changed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763